samples/gameSample/headers/tui.hpp

```diff
@@ -5,10 +5,51 @@
 #include <vector>
 #include <string>
 
+/**
+ * @brief class for communication with I/O
+ * 
+ */
 struct TUI {

```

Comment:
Why not a namespace + free funcs?

Reply:
Plan to introduce some private static variables, for screen size keeping, background color, text color etc.

Reply:
Well you can just add those as static functions in the file with the implementation. You don't necessarily have to declare your privates in the header at all.

Reply:
I don't know what that means.
I don't really have a problem, I just don't see any benefit and I just think about this differently I guess (encapsulation on a module level).

Reply:
> Plan to introduce some private static variables, for screen size keeping, background color, text color etc.

Ah sorry I read this as "private static functions". Yeah keeping state there makes sense